### ISSUE TITLE
Icon is not showing title on keyboard focus, whereas working fine for mouse hover

### ISSUE THREAD

**Issue by harshvardhan3946** [OfficeDev/office-ui-fabric-react#10492]
### Environment Information

- **Package version(s)**: 6.182.1
- **Browser and OS versions**: chrome, windows 10

### Icon title is visible on mouse hover, whereas through keyboard it not.

https://codepen.io/harshvardhan3946/pen/MWgPvrp?&editable=true

#### Actual behavior:

Tooltip "title" which is available on mouse hovering is not available for keyboard users

#### Expected behavior:

Tooltip "title" which is available on mouse hovering should be available for keyboard users

**Comment by aneeshack4** [comment]
@harshvardhan3946 Hi, I'm able to see the Tooltip title through keyboarding too: 
![image](https://user-images.githubusercontent.com/4161791/65191205-13f6d000-da28-11e9-9fdd-835d565a78c0.png)
This is what I see on mouse hover - which is the same thing:
![image](https://user-images.githubusercontent.com/4161791/65191278-499bb900-da28-11e9-8bea-16a62f04da34.png)

You say Icon title first and then Tooltip title - are these separate things? Please clarify.

**Comment by harshvardhan3946** [comment]
We see hover card as well as icon tittle on mouse hover. When we use keyboard we get hover cards, but there is no way from keyboard to see title. Refer below screenshot.

Ideally for a keyboard user after hover card, on right arrow press, title should be shown (assuming this is what is called tooltip).

![image](https://user-images.githubusercontent.com/14051057/65214778-e0b84d80-dac8-11e9-8d44-f653d4ffa006.png)

**Comment by aneeshack4** [comment]
@harshvardhan3946 I'm not able to repro this - I don't see this title tooltip at all... are there any specific repro steps when we keyboard or mouse over this icon or anything that we need to do once the hover card shows up?

**Comment by harshvardhan3946** [comment]
@aneeshack4 When you hover over icon, you will see title tooltip as shown in screenshot. Just place mouse pointer over icon.

**Comment by aneeshack4** [comment]
@harshvardhan3946 Oh I see - so I believe this is because when you mouse over you're able to distinguish between the icon vs the link right above it. Both lead to the hovercard opening but only when you mouse over the icon does the tooltip show up with the hovercard - when you just mouse over the link above ("Chicago") the tooltip doesn't show. Keyboarding only seems to let us go to the link, not the icon which is why the hovercard without the tooltip is showing. @micahgodbolt I looked at Tooltip.base.tsx and there's no handling of Mouse or Touch events - any ideas on where this is done or if something needs to be added?

**Comment by micahgodbolt** [comment]
Is this a hover card problem then? Maybe something for @atneik to take a look at?

**Comment by atneik** [comment]
HTML Title attribute doesn't support keyboarding, its only for mouse hover. Hence, [not the best choice for accessibility](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title#Accessibility_concerns). You could use the Tooltip component instead. In [this codepen example](https://codepen.io/atneik/pen/wvwVwmj?editors=0010) when hovered/keyboarded to the cell in second column it shows both the Tooltip and the HoverCard.

**Comment by aneeshack4** [comment]
@harshvardhan3946 Have you tried what @atneik suggested?
